nss-day-cohort-39/tabloid-mvc-depech-mode
# Post implementation

### Changes
- Created Post and MyPost controller and implemented Index, Create, Edit, and Delete actions
- Updated Post to have a hard delete
- Updated Post repository to contain methods required for associated actions and have categories

### Steps to Test
- Go to post test and check the Index view
- Try and create a new post
- Edit your new post
- Delete the post and ensure it does not appear in the Index list
- Make sure that Posts only display if you have selected a date in the past and will populate in "my posts" otherwise
- Make sure that Posts and My Posts display in descending order by date
